lib/src/cmaf/ham/manifestParser.ts

```diff
@@ -7,8 +20,52 @@ async function readHLS(manifestUrl: string): Promise<string> {
     return response.text();
 }
 
-
-export async function m3u8toHam(url: string) {
+export async function m3u8toHam(url: string): Promise<Presentation> {
+    const hls: string = await readHLS(url);
+    const parsedM3u8 = parseM3u8(hls);
+    const playlists: PlayList[] = parsedM3u8.playlists;
+    const mediaGroupsAudio = parsedM3u8.mediaGroups?.AUDIO;
+    let audioSwitchingSets: SwitchingSet[] = [];
+    let audioTracks : AudioTrack[] = [];
+    let selectionSets: SelectionSet[] = [];
     
+    for (let audio in mediaGroupsAudio) {
+        for (let attributes in mediaGroupsAudio[audio]) {
+            let language = mediaGroupsAudio[audio][attributes].language;
+            audioSwitchingSets.push(new SwitchingSet(audio, AUDIO_TYPE, '', 0, language,[]));
+            audioTracks.push(new AudioTrack(audio, AUDIO_TYPE, '', 0, language, 0, 0, 0,[]));

```

Comment:
Why some hardcoded values when creating the `SwitchingSet` and the `AudioTrack`?